nucleus/pom.xml

```diff
@@ -124,6 +124,8 @@
 
         <glassfish-corba.version>4.0.1</glassfish-corba.version>
         <stax-api.version>1.0-2</stax-api.version>
+        <slf4j.version>1.7.21</slf4j.version>
```

Comment:
These were the dependencies used by bean-validator packaging module in hk2. I do not want to upgrade those and see different set of issues. We can upgrade to appropriate version of sl4j depending upon the version required by bean-validator 2.0 whenever we introduce bean validator 2.0 support.